File: Pertemuan 9/TugasKalkulator.java
import java.util.Scanner;

public class TugasKalkulator {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.print("Masukan nilai A = ");
        double nilaiA = scan.nextDouble();
        System.out.print("Masukan nilai B = ");
        double nilaiB = scan.nextDouble();

        double hasil1 = penjumlahan(nilaiA, nilaiB);
        System.out.println();
        System.out.println("Nilai A + B = " + hasil1);
        double hasil2 = pengurangan(nilaiA, nilaiB);
        System.out.println();
        System.out.println("Nilai A - B = " + hasil2);
        double hasil3 = perkalian(nilaiA, nilaiB);
        System.out.println();
        System.out.println("Nilai A * B = " + hasil3);
    }

    public static double penjumlahan(double nilaiA, double nilaiB) {
        double hasil1 = nilaiA + nilaiB;
        return hasil1;
    }
    public static double pengurangan(double nilaiA, double nilaiB) {
        double hasil2 = nilaiA - nilaiB;
        return hasil2;
    }
    public static double perkalian(double nilaiA, double nilaiB) {
        double hasil3 = nilaiA * nilaiB;
        return hasil3;
    }
}

File: Pertemuan 9/tugaskalkulator.py
A = input("Masukkan nilai A = ")
A = float(A)
B = input("Masukkan nilai B = ")
B = float(B)

def penjumlahan(A, B):
    hasil1 = A + B
    return str(hasil1)

def pengurangan(A, B):
    hasil2 = A - B
    return str(hasil2)

def perkalian(A, B):
    hasil3 = A * B
    return str(hasil3)

print("Nilai A + B = " + penjumlahan(A, B)) 
print("Nilai A - B = " + pengurangan(A, B))
print("Nilai A * B = " + perkalian(A, B))